core/iwasm/aot/aot_runtime.c

```diff
@@ -1102,6 +1102,18 @@ aot_instantiate(AOTModule *module, bool is_sub_inst, uint32 stack_size,
     }
 #endif
 
+#if WASM_ENABLE_WASI_NN != 0
+    if (!is_sub_inst) {
+        if (!((AOTModuleInstanceExtra *)module_inst->e)->wasi_nn_ctx =
+                wasi_nn_initialize()))
+            {
```

Comment:
Coding guideline issue, should move `{` to the end of last line

Reply:
ok, I missed a parenthesis. I usually push to the upstream not for it to being reviewed but because my methodology. I forget that you check the code almost instantly :pray: